common/ethnicities/wc_ethnicities_troll.txt

```diff
@@ -0,0 +1,668 @@
+        1 = { name = mouth_tusks    range = { 1 1	  }    }
+    }
+	static_tusks = {
+        1 = { name = troll_tusks    range = { 0 1	  }    }
+    }
+	
+	###Fingers
+	gene_bs_fingers = {
+        1 = { name = fingers_troll    range = { 1 1	  }    }
+    }
+	
+	#Body
+	gene_bs_body_type = {
+		10 = { name = body_fat_head_fat_low		range = { 0.4 0.5 } }
+	}
+	
+	###Other	
+	hairstyles = {
+		1 = { name = troll_hairstyles 		range = { 0.0 1.0 } }
+	}
+	beards = {
+		1 = { name = no_beard 		range = { 0.0 1.0 } }
+	}
+	special_eyebrows = {
+        1 = {  name = no_eyebrows             range = { 0 1 }     }
+	}
+	face_detail_temple_def = {
+        1 = {  name = temple_def             range = { 0.2 0.8 }     }
+	}
+	expression_brow_wrinkles = {
+        1 = {  name = brow_wrinkles_01             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_02             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_03             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_04            range = { 0.5 1 }     }
+	}
+	expression_eye_wrinkles = {
+        1 = {  name = eye_wrinkles_01             range = { 0 1 }     }
+		1 = {  name = eye_wrinkles_02             range = { 0 1 }     }
+		1 = {  name = eye_wrinkles_03             range = { 0 1 }     }
+	}
+	expression_forehead_wrinkles = {
+        1 = {  name = forehead_wrinkles_01             range = { 0.1 1 }     }
+		1 = {  name = forehead_wrinkles_02             range = { 0.1 1 }     }
+		1 = {  name = forehead_wrinkles_03             range = { 0.1 1 }     }
+	}
+	complexion = {
+        1 = {  name = complexion_1             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_3             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_4             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_5             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_6             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_7             range = { 0.43 0.43 }     }
+	}
+}
+
+blood_troll_ethnicity = {
+	template = "darkspear_trollish"
+	
+	skin_color = {
+		20 = { 0.675 0.176 0.675 0.176 } # Pale Yellow/White
+		5 = { 0.392 0.169 0.392 0.169 } # Pale Green
+		1 = { 0.667 0.318 0.667 0.318 } # Pale Blue
+		5 = { 0.502 0.012 0.502 0.012 } # Brown/Gray
+	}
+	eye_color = {
+        1 = { 0 1 0 1 } # Black
+	}
+	hair_color = {
+		1 = { 1 0 1 0 } # Black/Gray
+		5 = { 0.129 0.043 0.137 0.110 } # Platinum Blonde & White
+		1 = { 0.102 0.886 0.102 0.886 } # Dark Brown
+		1 = { 0.153 0.875 0.153 0.875 } # Lighter Brown
+	}
+	
+	#Eyes
+	special_eyes = {
+        10 = { name = magic_eyes      range = { 0 1 } }
+    }
+	
+	#Body
+	gene_bs_body_type = {
+		10 = { name = body_fat_head_fat_low		range = { 0.149 0.255 } }
+	}
+	gene_bs_body_shape = {
+		10 = { name = body_shape_average_clothed		range = { 0 0 } }
+	}
+	
+	###Other
+	hairstyles = {
+		10 = { name = troll_hairstyles 		range = { 0.0 1.0 } }
+		1 = { name = no_hairstyles 		range = { 0.0 1.0 } }
```

Comment:
You can remove it. The baldness is handled by `bald_modifier` in `Warcraft-Guardians-of-Azeroth-2\common\scripted_modifiers\wc_portrait_modifiers.txt`. Do you want the blood trolls to be balder than other trolls?

Reply:
Yes, blood trolls should have a higher chance of being generated as bald.

Reply:
> 
> 
> You can remove it. The baldness is handled by `bald_modifier` in `Warcraft-Guardians-of-Azeroth-2\common\scripted_modifiers\wc_portrait_modifiers.txt`. Do you want the blood trolls to be balder than other trolls?

Removed these entries in 8bdb0de6d307090a2ef2ba00f1afc05f100fc3fc.